src/server/pages/user-page.jsx

```diff
@@ -23,7 +26,81 @@ h2 {
 }
 `
 
-export default class HomePage extends React.Component {
+class UserNotebookList extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = { notebookList: this.props.notebookList }
+    this.deleteNotebook = this.deleteNotebook.bind(this)
+  }
+
+  deleteNotebook(nbID) {
+    const notebookList = this.state.notebookList.filter(nb => nb.id !== nbID)
+    this.setState({ notebookList })
+  }
+
+  render() {
+    if (!this.state.notebookList.length) {
+      return <UserPageWithoutNotebooksPlaceholder isUserAccount={this.props.isUserAccount} />
+    }
+    return (
+      <React.Fragment>
+        <h2>
+            notebooks
+        </h2>
+        <NewNotebookButton />
+        <Table>
+          <thead>
+            <tr>
+              <th>title</th>
+              <th>last saved</th>
+              <th />
+              <th />
+            </tr>
+          </thead>
+          <tbody>
+            {this.state.notebookList.map(notebook => (
+              <tr key={notebook.id}>
+                <td><a href={`/notebooks/${notebook.id}/`}>{notebook.title}</a></td>
+                <td>{notebook.last_revision.slice(0, 19)}</td>
+                <td><a href={`/notebooks/${notebook.id}/revisions/`}>revisions</a></td>
+                <td>
```

Comment:
Showing as many delete buttons as there are notebooks seems a little heavy. I wonder if we'd do better to have a menu here hidden under a '...' button with options to show the revisions and delete?

Reply:
Yeah, that was my initial instinct as well. Might try another pass at an activity menu that includes a delete button after this PR lands.